CommentUpdater/src/main/java/org/eupathdb/sitesearch/data/comments/ProcessingResult.java

```diff
@@ -2,9 +2,24 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.stream.Collectors;
 
 class ProcessingResult {
-  List<DocumentInfo> toUpdate    = new ArrayList<>();
+  private final static String NL = System.lineSeparator();  
+
+  
+  List<SolrDocument> toUpdate    = new ArrayList<>();
   List<RecordInfo> toPostProcess = new ArrayList<>();
+  
+  public String toString() {
+    StringBuffer buf = new StringBuffer();
```

Comment:
* Why `StringBuffer` instead of `StringBuilder`?  Not important, but StringBuffer is slower than StringBuilder due to StringBuffer's synchronization